packages/wrangler/src/__tests__/pages.test.ts

```diff
@@ -1,56 +1,23 @@
-/* eslint-disable no-shadow */
```

Comment:
nit: Any reason we shouldn't put all Pages tests in the `pages` directory? If I'm looking for Pages tests, I might not notice that I need to check two places. I noticed D1 also has all of it's test files in the `d1` directory, so it'd be nice to be consistent with that as well.

Reply:
✅